### PR TITLE
Checksum on flush

### PR DESCRIPTION
**Pull request by sebn**
Needs https://github.com/cozy-labs/cozy-desktop/pull/822

### CODE REVIEW

**Comment by sebn** [comment]
Needs https://github.com/cozy-labs/cozy-desktop/pull/822

File: cli/src/local/event_buffer.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* @flow */
 
 type EventBufferMode = 'idle' | 'timeout'
-type FlushCallback<EventType> = (EventType[]) => void
+type FlushCallback<EventType> = (EventType[]) => any
```
Comment (aenario) [inline cli/src/local/event_buffer.js]: why this change ?

Comment (sebn) [reply cli/src/local/event_buffer.js]: Needed to make `LocalWatcher#handleEvents()` async and use `await` statements in it.
Otherwise `yarn flow` complains about the return type.

Comment (sebn) [reply cli/src/local/event_buffer.js]: It could also turn out to be useful in future tests.
